src/test/java/org/gridsuite/modification/server/modifications/LineAttachToNewVoltageLevelTest.java

```diff
@@ -38,8 +40,15 @@ private VoltageLevelCreationInfos getNewVoltageLevel() {
                 .equipmentName("NewVoltageLevel")
                 .nominalVoltage(379.3)
                 .substationId("s1")
-                .busbarSections(List.of(new BusbarSectionCreationInfos("v1bbs", "BBS1", 1, 1)))
-                .busbarConnections(List.of())
+                .nominalVoltage(379.1)
```

Comment:
useless ... nominalVoltage is set just above ...

Reply:
Done